Examples/php/callback/runme.php

```diff
@@ -2,8 +2,6 @@
 
 # This file illustrates the cross language polymorphism using directors.
 
-require("example.php");
-
```

Comment:
For now at least, I think we need to keep generating (and requiring/including) the PHP wrapper script.  It will probably just be an empty script (or perhaps only include the magic to try to load the extension, but that's increasingly not useful as all the mechanisms it can use are disabled by default except in the CLI version of PHP).  But by keeping the script we can have the same code working with currently generated wrappers and with your new style wrappers.